### PR TITLE
BayDAG All Contributions

### PR DESCRIPTION
**Pull request by dhensle**
One master PR that merges in the other BayDAG pull requests.  Used to test the SANDAG ABM3 example.

### CODE REVIEW

File: activitysim/abm/models/auto_ownership.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def auto_ownership_simulate(
     state: workflow.State,
     households: pd.DataFrame,
     households_merged: pd.DataFrame,
+    persons_merged: pd.DataFrame,
```
Comment (jpn--) [inline activitysim/abm/models/auto_ownership.py:40]: Seems like `persons_merged` isn't used in auto ownership, why are we adding it here?

**Review by jpn-- — COMMENTED** [review]
Only one minor change (and updating to clear the very simple conflicts against now-current main), then this looks clear to merge.

File: activitysim/abm/models/auto_ownership.py (hunk 1 above)
Comment (dhensle) [reply activitysim/abm/models/auto_ownership.py:40]: It is actually used in the preprocessor settings in SANDAG's ABM3 [here](https://github.com/ActivitySim/sandag-abm3-example/blob/main/configs/resident/auto_ownership_preprocessor.csv).  The issue is that the persons_merged table needs to include the disaggregate accessibility outputs.  Including the persons_merged table here in the call forces the person_merged table to be re-merged to include the disaggregate accessibility columns.  This wouldn't ordinarily be needed, but for ABM3 the auto_ownership model is the first model in the list so the persons_merged table wasn't yet updated.  

I am open to a different solution here if you have a better way to handle this.

Comment (jpn--) [reply activitysim/abm/models/auto_ownership.py:40]: Can we just add a comment in the code next to it that it's needed in the args list for the side effects?

I'm not a fan of the whole side effects system, but I don't have a better solution here that's easy.

Comment (dhensle) [reply activitysim/abm/models/auto_ownership.py:40]: Made issue #853 and referenced it in the code comment.

**Pull request merged**